packages/components/src/tag/tag.tsx

```diff
@@ -0,0 +1,88 @@
+import { Stack, styled } from '@tamagui/core'
+
+import { Text } from '../text'
+
+import type { TextProps } from '../text'
+
+type Props = {
+  size: 32 | 24
+  emoji?: string
+  label?: string
+  selected?: boolean
+  disabled?: boolean
+}
+
+// todo?: export this map from text
```

Comment:
follow-up

Reply:
Mentioned in description, currently used here, in button, context tag and in tabs.